README.md

```diff
@@ -1,5 +1,98 @@
 maven-plugin
 ============
-
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/maven-plugin)](https://plugins.jenkins.io/maven-plugin)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/maven-plugin?label=changelog)](https://github.com/jenkinsci/maven-plugin/blob/master/CHANGELOG.md)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/maven-plugin?color=blue)](https://plugins.jenkins.io/maven-plugin)
 Jenkins plugin for building Maven 2/3 jobs.
 See [Maven Project Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Maven+Project+Plugin) on Jenkins' Wiki for more information.
+
+Older versions of this plugin may not be safe to use. Please review the
```

Comment:
The security section shouldn't be here, the plugin site will automatically populate it from the update center API

Reply:
@olamy this shouldn't be here, the information is duplicated and requires manual maintenance and syncing, the plugin site will handle it for you